public/docs/ts/latest/tutorial/toh-pt6.jade

```diff
@@ -0,0 +1,252 @@
+include ../_util-fns
+
+:marked
+  # Http
+  
+  Our application has become a huge success and our stakeholders have already expanded the vision to include integration with a hero api. 
+  
+  The current solution limits us to a fixed set of heroes, but integration with the api will allow us to load heroes dynamically from a database. We will also be able to add, edit and delete heroes. 
```

Comment:
we're not using a database, and I don't think that is important yet. We are getting our heroes from a web server api. This will allow us to get the data asynchronously and we can easily see how we can expand this to hit any database we want. I think it's worth rewording to something like that.